pilot/pkg/kube/inject/testdata/webhook/daemonset.yaml.injected

```diff
@@ -48,10 +49,7 @@ spec:
         - "15020"
         - --applicationPorts
         - "80"
-        ports:
-        - name: http-envoy-prom
-          containerPort: 15090
-          protocol: TCP
+        - --identity-domain=cluster.local
```

Comment:
Can you give another name?

Reply:
I would really like to. See the other discussions. But the citadel already has this parameter and it doesn't make sense to use a different one. I would like to rename all to "trust-domain" in a later PR, but discussions are ongoing.